### ISSUE TITLE
The default X-Rate-Limit-Reset value format is not culture invariant

### ISSUE THREAD

**Issue by martincostello** [stefanprodan/AspNetCoreRateLimit#2]
The `X-Rate-Limit-Reset` response header should allow for configurability.

Using the default configuration from the documentation for local testing, I get a value similar to the following in the HTTP response headers (a date one week from 14:07 BST today when the application was started):

`x-rate-limit-reset: 06/10/2016 13:07:24`

Given I'm running on a en-GB edition of Windows 10, this would suggest that the localisation settings of the host are contributing to the formatting of the value written by the middleware due to `ToString()` being used instead of with a specific format and culture ([here](https://github.com/stefanprodan/AspNetCoreRateLimit/blob/e9072dd0a61fd010ac91f3c428b8abdbc5848c15/src/AspNetCoreRateLimit/Core/RateLimitCore.cs#L93) and [here](https://github.com/stefanprodan/AspNetCoreRateLimit/blob/e9072dd0a61fd010ac91f3c428b8abdbc5848c15/src/AspNetCoreRateLimit/Core/RateLimitCore.cs#L99)).

By default the invariant culture should be used to ensure consistent behaviour (as [`RetryAfterFrom()`](https://github.com/stefanprodan/AspNetCoreRateLimit/blob/e9072dd0a61fd010ac91f3c428b8abdbc5848c15/src/AspNetCoreRateLimit/Core/RateLimitCore.cs#L107-L113) appears to), but also allow for the format to be configured by the user.

For example, if a client wishes to programmatically determine when they can retry from, a format such as the `u` pattern (`yyyy'-'MM'-'dd HH':'mm':'ss'Z'`) would be easier to parse and as it contains the time zone in the value as is thus less ambiguous. Alternatively, an integrator may wish to return the value as a number of seconds instead of as a timestamp.

**Comment by stefanprodan** [comment]
Thanks for reporting this issue. I've chosen to use UTC and the `"o"` format so the date can be deserialized with:

```cs 
DateTime resetDate = DateTime.ParseExact(resetHeader, "o", DateTimeFormatInfo.InvariantInfo);
```

**Comment by martincostello** [comment]
Fixed in 1.0.2 so closing.
